abnf/odata-abnf-testcases.yaml

```diff
@@ -2456,6 +2497,66 @@ TestCases:
     Rule: select
     Input: $select=*
 
+  - Name: 5.1.3 Select - $count
+    Rule: select
+    Input: $select=Sizes($count=true)
```

Comment:
Can I also write this as Sizes, Sizes/$count?

Reply:
Do you want to add a test case for the alternative form with a comment calling it out?

Reply:
Added test.